tests/functional/services/policy_engine/test_vulnerability_scanner.py

```diff
@@ -0,0 +1,134 @@
+import json
+import logging
+
+import pytest
+
+import tests.functional.services.policy_engine.utils.api as policy_engine_api
+from tests.functional.services.utils import http_utils
+
+
+@pytest.mark.parametrize(
+    "image_digest",
+    [
+        "sha256:80a31c3ce2e99c3691c27ac3b1753163214494e9b2ca07bfdccf29a5cca2bfbe",  # alpine-test
+        "sha256:406413437f26223183d133ccc7186f24c827729e1b21adc7330dd43fcdc030b3",  # debian-test
+        "sha256:fe3ca35038008b0eac0fa4e686bd072c9430000ab7d7853001bde5f5b8ccf60c",  # centos-test
+    ],
+)
+class TestVulnerabilityScanner:
```

Comment:
Nit: The test methods in this class are sort of hard to read because there are no line breaks separating logical groupings of lines of code.

Reply:
Also, same thought as above about adding comments.

Reply:
fixed